debian/tvheadend.postinst

```diff
@@ -10,16 +10,15 @@ configure)
 
    if ! getent passwd $HTS_USER >/dev/null; then
         echo >&2 "Creating user: $HTS_USER..."
-        adduser --quiet --system --group --shell /bin/bash $HTS_USER
+        adduser --quiet --system --group --home /var/lib/tvheadend $HTS_USER
```

Comment:
what about migration? Maybe add a check 'if old, inform user of new setup?

Reply:
This won't get executed on existing installs, so there is no migration. As it isn't a benefit on existing Installations I'd assume that it can stay as-is.

Reply:
This particular bit of code only executes on new installs, not upgrades.  Detecting when such an upgrade occurred would be potentially problematic, so I didn't go off into those weeds.  Plus, the code you added already has a warning when the old configuration directory is in use.

Reply:
so true :) sorry for the noise